src/DotNest.Core.SDK.Web/Program.cs

```diff
@@ -13,12 +13,8 @@
         .AuthorizeApiRequestsIfEnabled(builder.Configuration)
         .AllowMiniProfilerOnAdmin()
         .AddDatabaseShellsConfigurationIfAvailable(configuration)
-        .ConfigureSmtpSettings(overrideAdminSettings: false);
-
-    if (builder.Environment.IsDevelopment())
-    {
-        orchardCoreBuilder.AddSetupFeatures("OrchardCore.AutoSetup");
-    }
+        .ConfigureSmtpSettings(overrideAdminSettings: false)
+        .AddSetupFeatures("OrchardCore.AutoSetup");
```

Comment:
Why did you delete the condition for autosetup?

Reply:
I'll revert it. I was doing some highly qualified trial and error to fix something.